mod/chain-spec/pkg/chain/chain_spec.go

```diff
@@ -476,3 +478,9 @@ func (c chainSpec[
 ]) GetCometBFTConfigForSlot(_ SlotT) CometBFTConfigT {
 	return c.Data.CometValues
 }
+
+func (c chainSpec[
```

Comment:
comment `// GetValidatorSetCap retrieves the maximum number of validators allowed in the active set.`